br/pkg/lightning/restore/checksum_test.go

```diff
@@ -281,56 +282,56 @@ func (s *checksumSuite) TestGcTTLManagerSingle(c *C) {
 	}()
 
 	err := manager.addOneJob(ctx, "test", uint64(time.Now().Unix()))
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 
 	time.Sleep(2*time.Second + 10*time.Millisecond)
 
 	// after 2 seconds, must at least update 5 times
 	val := atomic.LoadInt32(&pdClient.count)
-	c.Assert(val, GreaterEqual, int32(5))
+	require.GreaterOrEqual(t, val, int32(5))
 
 	// after remove the job, there are no job remain, gc ttl needn't to be updated
 	manager.removeOneJob("test")
 	time.Sleep(10 * time.Millisecond)
 	val = atomic.LoadInt32(&pdClient.count)
 	time.Sleep(1*time.Second + 10*time.Millisecond)
-	c.Assert(atomic.LoadInt32(&pdClient.count), Equals, val)
+	require.Equal(t, val, atomic.LoadInt32(&pdClient.count))
```

Comment:
Can you help me change ```count``` type to uber/atomic? because it is easy to forget to use the atomic operation to read or modify the variables.

Reply:
ok